cmd/tk/export.go

```diff
@@ -70,11 +79,21 @@ func exportCmd() *cli.Command {
 			if err := tmpl.Execute(&buf, m); err != nil {
 				log.Fatalln("executing name template:", err)
 			}
-			name := strings.Replace(buf.String(), "/", "-", -1)
+
+			// Replace all os.path separators in string in order to not accidentally create subfolders
+			name := strings.Replace(buf.String(), string(os.PathSeparator), "-", -1)
```

Comment:
Wondering if this should be `os.PathSeparator` or plain `/`, because apiVersion always includes `/` and on Windows it would be persisted (whatever happens then) and on UNIX it would become a dash

Reply:
I would opt for `os.PathSeparator` as this really than only injects subfolders if intended by the user